functions/index.js

```diff
@@ -4,35 +4,27 @@ const { getGeocodes } = require("./middleware");
 exports.getGeo = functions.https.onRequest(async (request, response) => {
   const address = request.query.address;

```

Comment:
before can we check if request.query is not empty

```
Object.keys(req.query).length === 0 
return 400
```

Reply:
You're right. Instead of checking if it's undefined, we can check if it exists and if it does, we can check to see if it's empty as well. Our condition would be `(if non existent OR if empty) return 400`